### PR TITLE
add property-cached

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://pypi.org/project/property-cached/

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge/staged-recipes this is ready for review.

File: recipes/property-cached/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - property_cached
+
+about:
+  home: The package home page
```
Comment (beckermr) [inline recipes/property-cached/meta.yaml]: You should set this.

Comment (ocefpaf) [reply recipes/property-cached/meta.yaml]: Done!

File: recipes/property-cached/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - property_cached
+
+about:
+  home: https://github.com/althonos/property-cached/
```
Comment (isuruf) [inline recipes/property-cached/meta.yaml]: Shouldn't this be https://github.com/pydanny/cached-property ?

File: recipes/property-cached/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - setuptools
```
Comment (isuruf) [inline recipes/property-cached/meta.yaml]: Why do you need this at runtime?

Comment (ocefpaf) [reply recipes/property-cached/meta.yaml]: Like I it says in the commit message, it uses `pkg_resources` :wink:

File: recipes/property-cached/meta.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipes/property-cached/meta.yaml]: Interesting, that was in the PyPI metadata. Let's point to pydanny.

**Comment by ocefpaf** [comment]
Looks loike I missed https://github.com/conda-forge/cached-property-feedstock/

**Comment by ocefpaf** [comment]
Actually they are not the same! Re-opening.

File: recipes/property-cached/meta.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipes/property-cached/meta.yaml]: It is https://github.com/althonos/property-cached/, a fork, different name, and new release.

**Comment by ocefpaf** [comment]
@beckermr is this good to go?

**Pull request merged**